src/containers/BlockedList/index.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import Proptypes from 'prop-types';
+import { Table } from 'antd';
+import { DeleteFilled } from '@ant-design/icons';
+import Button from 'components/Button';
+import Container from 'components/Container';
+import Header from 'components/Header';
+import Modal from 'components/Modal';
+import FormModal from './components/FormModal';
+
+import styles from './index.module.scss';
+
+const BlockedList = ({ data, onUpdateClient }) => {
+  const [addModal, setAddModal] = useState(false);
+  const [deleteModal, setDeleteModal] = useState(false);
+  const [activeMac, setActiveMac] = useState({});
+
+  const addClient = ({ macAddress }) => {
+    const details = {
+      blocklistDetails: { enabled: true },
+      model_type: 'ClientInfoDetails',
+    };
+    onUpdateClient(macAddress, details);
+    setAddModal(false);
+  };
+
+  const deleteClient = () => {
+    const details = {
+      blocklistDetails: { enabled: false },
+      model_type: 'ClientInfoDetails',
+    };
+    onUpdateClient(activeMac.macAddress, details);
+    setDeleteModal(false);
+  };
+
+  const columns = [
+    {
+      title: 'MAC ADDRESS',
+      dataIndex: 'macAddress',
+      key: 'mac',
+      width: 900,
+    },
+    {
+      title: '',
+      dataIndex: '',
+      key: 'deleteModel',
+      width: 60,
+      render: (_, record) => (
+        <Button
+          title={`delete-mac-${record.macAddress}`}
+          className={styles.InfoButton}
+          type="danger"
+          icon={<DeleteFilled />}
+          onClick={() => {
+            setActiveMac({ ...record });
+            setDeleteModal(true);
+          }}
+        />
+      ),
+    },
+  ];
+
+  return (
+    <Container>
+      <Modal
+        onCancel={() => setDeleteModal(false)}
+        onSuccess={deleteClient}
+        visible={deleteModal}
+        title="Are you sure?"
+        buttonText="Delete"
+        buttonType="danger"
+        content={
+          <p>
+            Are you sure you want to delete the MAC Address:{' '}
```

Comment:
this message does not reflect what you are doing. "Are you sure you want to remove the Client: {MAC} from the Blocked List?"